pom.xml

```diff
@@ -1345,7 +1492,7 @@
           -->
           <excludedPages>
             <excludedPage>dependencies.html</excludedPage>
-            <excludedPage>cobertura/**</excludedPage>
+            <excludedPage>jacoco/**</excludedPage>
```

Comment:
confirmed, as plugin generate link that is not 100% valid, links check gives `http://www.jacoco.org/jacoco: 301 Moved Permanently` . NO `/` at the end of url.